.vscode/settings.json

```diff
@@ -1,6 +1,12 @@
 {
+  "python.formatting.provider": "none",
```

Comment:
I think this needs to be removed: 

![image](https://github.com/arguflow/arguflow/assets/15804464/49594aa4-3a87-411b-aeac-7f5b36d0a84a)